src/index.d.ts

```diff
@@ -235,26 +235,80 @@ export type WebSocketStatus =
     | 'RECONNECTING'
     | 'CONNECTED';
 
-export class Shard {
+export interface ShardCommands {
+    'auth': [token: string]
+    'send stats': undefined
+    'send logs': undefined
+    'set state': [state: PowerState]
+    'send command': [command: string]
+}
+
+export interface ServerStats {
```

Comment:
The object keys here should be in camelCase (can be changed with caseConv util).

Reply:
These strings are from panel api. Do you suggest creating a map and "translate" commands for panel?

Reply:
`src/util/caseConv.js` can be used for conversion the packetHandler file.

Reply:
Sure, will do in a moment